city_scrapers/spiders/chi_state_university.py

```diff
@@ -0,0 +1,200 @@
+                        link = meeting.xpath(".//a").attrib["href"]
+                        dateStr = meeting.xpath(".//text()").extract()[0]
+                        dateStr = re.sub(",", " ", dateStr)
+                        dateStr = re.sub(r"\s+", " ", dateStr)
+
+                        commonStrings = [
+                            "Standing Committee",
+                            "Special Board",
+                            "Full Board",
+                        ]
+
+                        for i in commonStrings:
+                            if i in title:
+                                title = i
+
+                        minutes_map[title + " " + dateStr].append(
+                            {
+                                "title": title + " " + dateStr,
+                                "href": "https://www.csu.edu" + link,
+                            }
+                        )
+            return minutes_map
+
+    def _parse_meetings(self, response):
+        for data in response.xpath("/html/body/div/div[4]/div/div[1]/div/div"):
+            for label in data.xpath(".//label"):
+                title = label.xpath(".//text()").extract()[0]
+                sibling = label.xpath("following-sibling::*")
+                self._validate_location(sibling)
+                for item in sibling.xpath(".//li"):
+                    months = {m.lower() for m in month_name[1:]}
+                    text = item.xpath(".//text()").extract()[0]
+                    monthMatch = next(
+                        (word for word in text.split() if word.lower() in months), None
+                    )
+                    if monthMatch is None:
+                        continue
+                    start = self._parse_start(item)
+
+                    meeting = Meeting(
+                        title=title,
+                        description=self._parse_description(item),
+                        classification=self._parse_classification(title),
+                        start=start,
+                        end=None,
+                        all_day=False,
+                        time_notes=self._parse_time_notes(item),
+                        location=self._parse_location(item),
+                        links=self._parse_links(item, title, start),
+                        source=self._parse_source(response),
+                    )
+                    meeting["status"] = self._get_status(meeting)
+                    meeting["id"] = self._get_id(meeting)
+
+                    yield meeting
+
+    def _parse_description(self, item):
+        parts = item.xpath(".//text()").extract()
+        description = "".join([str(x) for x in parts])
+        description = (
+            description.replace("\xa0", " ")
+            .replace("\n", " ")
+            .replace("\t", " ")
+            .strip()
+        )
+        description = re.sub(r"\s+", " ", description)
+        return description
+
+    def _parse_classification(self, title):
```

Comment:
Classification should always return one of the options, so we can default to `BOARD` here unless it's a committee